jax/_src/numpy/lax_numpy.py

```diff
@@ -4773,15 +4773,14 @@ def tril(m: ArrayLike, k: int = 0) -> Array:
   JAX implementation of :func:`numpy.tril`
 
   Args:
-    m: input array.
+    m: input array. Must have ``m.ndim >= 2``.
     k: k: optional, int, default=0. Specifies the sub-diagonal above which the
       elements of the array are set to zero. ``k=0`` refers to main diagonal,
       ``k<0`` refers to sub-diagonal below the main diagonal and ``k>0`` refers
       to sub-diagonal above the main diagonal.
 
   Returns:
-    An array containing the lower triangle of the given array with elements above
-    the sub-diagonal specified by ``k`` are set to zero.
+    An array with the same shape as the input
```

Comment:
I meant here that you should keep the original description, but also mention the shape of the output.

Reply:
Sorry, I misunderstood it. Changed it.